lib/src/native/realm_core.dart

```diff
@@ -1010,6 +1010,15 @@ class _RealmCore {
     });
     return completer.future;
   }
+  
+  UserHandle? getCurrentUser(AppHandle appHandle) {
+    final userPtr = _realmLib.realm_app_get_current_user(appHandle._pointer);
```

Comment:
We can't use `_realmLib.invokeGetPointer` here, but should we check for error somehow?

Reply:
Some functions that return pointers can't fail. This one and http_transport_new are such example. We just use the function directly in these cases.